calico-enterprise/getting-started/bare-metal/about.mdx

```diff
@@ -132,22 +108,18 @@ To learn how to restrict traffic to/from hosts and VMs using Calico network poli
 
     You can configure the Calico node by tuning the environment variables defined in the `/etc/calico/calico-node/calico-node.env` file. For more information, see the [Felix configuration reference](../../reference/resources/felixconfig.mdx).
 
-### Configure hosts to communicate with your Kubernetes cluster
-
-You must ensure that the non-cluster hosts or VMs can communicate with your Kubernetes cluster. If you are using an ingress controller or an external load balancer, ensure it is configured to allow TCP Layer 4 passthrough. This is required for non-cluster host agent to establish a mutual TLS (mTLS) connection to the cluster via the `typhaEndpoint` endpoint.
-
-Below are some vendor-specific considerations:
```

Comment:
Vendor-specific considerations were removed because they are outdated and may not accurately represent how customers configure their environments.